eng/pipelines/azuresdkpartnerdrops-to-devops.yml

```diff
@@ -1,22 +1,27 @@
 parameters:
-- name: PartnerDropsBlobSourceSuffix
+- name: BlobPath
   type: string
-  default: ''
+  default: '<team>/dotnet/<version>'
```

Comment:
Should we remove the default since it seems like a placeholder that wouldn't be expected to work? Or is this a syntax that gets parsed downstream?

Reply:
We just added it to make it more intuitive for people who will use the pipeline. So right now it's more like an example.

Reply:
Yes this is just a placeholder to give something for people to follow.